src/ExpenseTrackerGrupo2/Business/Services/Interfaces/IUserService.cs

```diff
@@ -0,0 +1,6 @@
+namespace ExpenseTrackerGrupo2.Business.Services.Interfaces;
+
+public interface IUserService
+{
+    // TODO: Complete according external service
```

Comment:
Why interface its empty?
![Screenshot from 2024-09-24 09-53-49](https://github.com/user-attachments/assets/67cbd720-e90d-419d-aaab-61a13ff3c172)

Reply:
Actually, it's empty due to we won't define our external service. That's why I left it empty.

Reply:
So the UML needs an update.

Reply:
Could you develop your idea further?

Reply:
We'll work on it later @alex-paca-jalafund. We need to keep moving on. I suggest we can come back later according the external service.

Reply:
mm... I dont agree, but services has to be worked asap, so yep lets keep moving.